doc/changes/changes-4.10.xml

```diff
@@ -0,0 +1,312 @@
+overhead for running these immediate methods and applying their results can
+become a major factor in the runtime of complex computations that create
+thousands or millions of objects.
+<P/>
+
+To address this, various steps were taken:
+<List>
+<Item>some immediate methods were turned into regular methods;</Item>
+<Item>a special handlers for <C>SetSize</C> was created that performs deductions which previously were taken care of by immediate methods;</Item>
+<Item>some immediate methods were replaced by implications (set via <C>InstallTrueMethod</C>), a mechanism that essentially adds zero overhead, unlike immediate methods;</Item>
+<Item>various group constructors were modified to precompute and preset properties of freshly created group objects, to avoid triggering immediate methods for these.</Item>
+</List>
+
+As a result of these and other changes, consider the following example;
+with GAP 4.9, it takes about 130 seconds on one test system, while with
+GAP 4.10 it runs in about 22 seconds, i.e., more than six times faster.
+<Log><![CDATA[
+G:=PcGroupCode( 741231213963541373679312045151639276850536621925972119311,11664);;
+IsomorphismGroups(G,PcGroupCode(CodePcGroup(G),Size(G)))<>fail;
+]]></Log>
+
+Relevant pull requests and issues: <URL><LinkText>&Hash;2386</LinkText><Link>https://github.com/gap-system/gap/pull/2386</Link></URL>, <URL><LinkText>&Hash;2387</LinkText><Link>https://github.com/gap-system/gap/pull/2387</Link></URL>, <URL><LinkText>&Hash;2522</LinkText><Link>https://github.com/gap-system/gap/pull/2522</Link></URL>.
+</Item>
+
+<!-- 
+ -->
+<Mark>Change definition of <C>IsPGroup</C> to <E>not</E> require finiteness</Mark><Item>
+
+This is a small change in terms of amount of code changed, but we list it here
+as it has a potential (albeit rather unlikely) impact on the code written by &GAP;
+users:
+In the past, the GAP manual entry for <C>IsPGroup</C> defined p-groups as being
+finite groups, which differs from the most commonly used definition for
+p-groups. Note however that there was not actual implication installed from
+<C>IsPGroup</C> to <C>IsFinite</C>, so it always was possible to actually created
+infinite groups in the filter <C>IsPGroup</C>.
+
+In GAP 4.10, we adjusted (in <URL><LinkText>&Hash;1545</LinkText><Link>https://github.com/gap-system/gap/pull/1545</Link></URL>)
+the documentation for <C>IsPGroup</C> to the commonly accepted definition for
+p-groups. In addition, code in the GAP library and in packages using
+<C>IsPGroup</C> was audited and (in a very few cases) adjusted to explicitly check
+<C>IsFinite</C> (see e.g. <URL><LinkText>&Hash;2866</LinkText><Link>https://github.com/gap-system/gap/pull/2866</Link></URL>).
+</Item>
+
+<!-- 
+ -->
+<Mark>Experimental support for using the Julia garbage collector</Mark><Item>
+
+It is now possible to use the garbage collector of the <URL Text="Julia language">https://julialang.org</URL>
+instead of GAP's traditional GASMAN garbage collector. This is partly
```

Comment:
Maybe `<Package>GASMAN</Package>` for a different font? (And maybe for Julia too).